pkg/web/static/openapi.json

```diff
@@ -2061,28 +2404,128 @@
+            "type": "string",
+            "description": "The name of the registered directory that contains the counterparty."
+          },
+          "protocol": {
+            "type": "string",
+            "description": "The protocol used to communicate with the counterparty."
+          },
+          "common_name": {
+            "type": "string",
+            "description": "The common name of the counterparty."
+          },
+          "endpoint": {
+            "type": "string",
+            "description": "The endpoint URL of the counterparty."
+          },
+          "travel_address": {
+            "type": "string",
+            "description": "The travel address of the counterparty."
+          },
+          "name": {
+            "type": "string",
+            "description": "The name of the counterparty."
+          },
+          "website": {
+            "type": "string",
+            "description": "The website URL of the counterparty."
+          },
+          "country": {
+            "type": "string",
+            "description": "The country of the counterparty."
+          },
+          "business_category": {
+            "type": "string",
+            "description": "The business category of the counterparty."
+          },
+          "vasp_categories": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The list of VASP categories that the counterparty belongs to."
+          },
+          "verified_on": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The date and time when the counterparty was verified."
+          },
+          "ivms101": {
+            "type": "string",
+            "description": "The IVMS101 certificate of the counterparty."
+          },
+          "created": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The date and time when the counterparty was created."
+          },
+          "modified": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The date and time when the counterparty was last modified."
           }
         }
       },
-      "CryptoAddress": {
-        "description": "CryptoAddress associated with account that needs to be added or updated",
-        "content": {
-          "application/json": {
-            "schema": {
-              "$ref": "#/components/schemas/CryptoAddress"
+      "CounterpartyList": {
+        "type": "object",
+        "description": "The response object for the counterparties endpoint.",
+        "properties": {
+          "page": {
+            "$ref": "#/components/schemas/PageQuery"
+          },
+          "counterparties": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Counterparty"
             }
           }
         }
+      },
+      "TravelAddress": {
```

Comment:
The `TravelAddress` object [here](https://github.com/trisacrypto/envoy/blob/main/pkg/web/api/v1/utilities.go#L8-L11) differs from the information displayed in this object. Would you mind taking another look?